src/core/ev_handler.rs

```diff
@@ -101,7 +103,7 @@ pub fn handle_event(
             }
             // Decrement the s_mark and get the preceeding index
             p.search_mark = p.search_mark.saturating_sub(1);
-            let y = p.search_idx[p.search_mark];
+            let y = *p.search_idx.iter().nth(p.search_mark).unwrap();
```

Comment:
Why did you change this? Isn't this functionally the same as before, just syntactically more complex?

Reply:
`PagerState::search_idx` is now a BTreeSet and hence to extract things from it, we just have to do this complex thing.

Reply:
Ah, gotcha; that makes sense.